.github/workflows/code-qa.yml

```diff
@@ -56,7 +73,15 @@ jobs:
       - name: Install dependencies
         run: npm run install:all
       - name: Run unit tests
-        run: npm test
+        working-directory: webview-ui
+        run: npx jest
+
+  unit-test:
```

Comment:
We have a branch rule dictating that "unit-test" must pass in order for the PR to be merged, hence this composite job.